networkx/drawing/tests/test_pydot.py

```diff
@@ -111,48 +111,46 @@ def test_pydot_issue_7581(tmp_path):
 
 def test_pydot_issue_258():
     G = nx.Graph([("Example:A", 1)])
-    with pytest.raises(ValueError):
-        nx.nx_pydot.to_pydot(G)
-    with pytest.raises(ValueError):
-        nx.nx_pydot.pydot_layout(G)
+    pydot_G = nx.nx_pydot.to_pydot(G)
+    # Confirm that nodes exist in a
+    # Keys as integers aren't preserved in the conversion. They are read as strings.
+    assert {str(n) for n in G.nodes}.issubset(nx.nx_pydot.from_pydot(pydot_G).nodes)
```

Comment:
I'm not sure what the intention would be for this test now (as colons in node names in `dot` [are ports](https://graphviz.org/docs/attr-types/portPos/); happy to adjust / restore the check or remove this part.

Reply:
I think this test is meant to address pydot issue 258 (based on its name) and since that issue has been addressed in pydot we should remove the test of it here. But I could be convinced otherwise.